front/app/modules/commercial/analytics/admin/containers/Proposals.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+// hooks
+import useFeatureFlag from 'hooks/useFeatureFlag';
+
+// components
+import ProposalsCard from '../components/ProposalsCard';
+import { StatCardPeriodProps } from '../typings';
+
+export default ({ ...props }: StatCardPeriodProps) => {
+  const proposalsActive = useFeatureFlag({ name: 'initiatives' });
+  const analyticsActive = useFeatureFlag({ name: 'analytics' });
```

Comment:
After Koen's comment on slack I think we can remove the analytics feature flag here- also did that for the Input status card